packages/account/src/Components/forms/form-fields.jsx

```diff
@@ -21,15 +21,15 @@ export const DateOfBirthField = ({ name, portal_id, ...rest }) => (
     </Field>
 );
 
-export const FormInputField = ({ name, optional = false, warn, ...rest }) => (
+export const FormInputField = ({ name, optional = false, ...rest }) => (
     <Field name={name}>
         {({ field, form: { errors, touched } }) => (
             <Input
                 type='text'
                 required={!optional}
                 autoComplete='off'
                 error={touched[field.name] && errors[field.name]}
-                warn={warn}
+                warn={rest.warn}
```

Comment:
what was the reason behind this change?

Reply:
we're spreading field props within the form and typescript was giving the error that `warn` is required as this file is still in `jsx`. Writing this way solves the required prop error